plugins/woocommerce/includes/wc-conditional-functions.php

```diff
@@ -517,7 +517,8 @@ function wc_is_file_valid_csv( $file, $check_path = true ) {
  * @since 6.0.0
  * @return bool
  */
-function wc_current_theme_is_fse_theme() {
+function wp_is_block_theme()
```

Comment:
We don't want to replace this. We want to add a deprecation here. You can use `wc_deprecated_function`